Extension/package.json

```diff
@@ -3309,67 +3309,116 @@
       "integrity": "6756365C82B5BD90C6A40C13CF8E3599E4A1DB50628111FFDB3A76FF85256BA4"
     },
     {
-      "description": "Mono Framework Assemblies",
-      "url": "https://go.microsoft.com/fwlink/?LinkId=2027135",
+      "description": "OpenDebugAD7 (Windows x86)",
+      "url": "https://download.visualstudio.microsoft.com/download/pr/65d2e4a8-b9e4-40e8-a131-51b3b6af7b43/de9fa59805117816e8a5f94c8266f08f/win7-x86.zip",
```

Comment:
Aren't we supposed to use fwlinks?

Reply:
Is there a reason to do so? If we need to ship a new version of the debugger, there will be a new link anyways.

We ship with the VS CDN in the C# extension: https://github.com/OmniSharp/omnisharp-vscode/blob/724c565ac5ba7276b2d0a54ca7204b396a44501e/package.json#L220